recommend/policy.go

```diff
@@ -214,8 +147,7 @@ func (img *ImageInfo) getPolicyFromImageInfo() {
 
 	err = createRuntimePolicy(img)
 	if err != nil {
-		log.WithError(err).Error("Failed to create runtime policy")
-		return
+		log.Info("No serviceaccount access detected. Skipping runtime policy creation on ", img.Distro)
```

Comment:
```suggestion
		log.Infof("Failed to create runtime policy creation for %s/%s/%s. err=%s", img.Namespace, img.Deployment, img.Name, err)
```
The child functions of `createRuntimePolicy()` throw error for things like conn failure or RPC call failure. So we should print it here.

In case of skipping the policy creation, we just log that in `checkProcessFileData()`. So we're good in that case.